packages/applicant/src/pages/api/routes/submissions/[submissionId]/sections/[sectionId]/questions/[questionId]/attachments/[attachmentId]/remove.test.tsx

```diff
@@ -0,0 +1,66 @@
+import handler from './remove.page';
+import { routes } from '../../../../../../../../../../../utils/routes';
+import { deleteAttachmentByQuestionId } from '../../../../../../../../../../../services/SubmissionService';
+
+jest.mock('../../../../../../../../../../../services/SubmissionService');
+jest.mock('../../../../../../../../../../../utils/jwt', () => ({
+  getJwtFromCookies: () => 'jwt',
+}));
+
+describe('handler', () => {
+  const testCases = [
+    {
+      description: 'without query string',
+      query: {},
+      expectedQueryString: '',
+    },
+    {
+      description: 'with fromCYAPage query string',
+      query: { fromCYAPage: true },
+      expectedQueryString: '?fromCYAPage=true',
+    },
+    {
+      description: 'with fromSubmissionSummaryPage query string',
+      query: { fromSubmissionSummaryPage: true },
+      expectedQueryString: '?fromSubmissionSummaryPage=true',
+    },
+  ];
+
+  test.each(testCases)(
```

Comment:
convention is to use `it` in this repo

```suggestion
  it.each(testCases)(
```